README.md

```diff
@@ -33,6 +33,16 @@
   <a href="https://snack.expo.io/@xcarpentier/gifted-chat" target="_blank"><i>demo</i></a>
 </p>
 
+## Sponsor
+
+<p align="center">
+ <sup>Sponsored by <a href="https://pusher.com/chatkit?utm_source=github&utm_campaign=react-native-gifted-chat-sponsorship">Pusher Chatkit</a>:</sup>
+</p>
+<p align="center">
+  <a href="https://pusher.com/chatkit?utm_source=github&utm_campaign=react-native-gifted-chat-sponsorship">
```

Comment:
Will need updating to `FaridSafi/react-native-gifted-chat` once merged. Ideally, this would be relative but I don't _think_ I can do that with an `img` element like I can with the equivalent GitHub markdown.